### PR TITLE
Verbose option

### PR DESCRIPTION
**Pull request by orthagonal**
branches from the es6 auto PR (the server.log statements that have to be changed are mostly inside the 'auto' call, this should be an easy merge after the 'es6' and 'auto' branch merges are done)

### CODE REVIEW

File: index.js.BASE.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
+'use strict';
```
Comment (jgallen23) [inline index.js.BASE.js]: this file should not be checked in

File: index.js.REMOTE.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
+'use strict';
```
Comment (jgallen23) [inline index.js.REMOTE.js]: :(

File: index.js.LOCAL.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
+'use strict';
```
Comment (jgallen23) [inline index.js.LOCAL.js]: :(

**Comment by orthagonal** [comment]
I switched to using the Github Desktop app last week.  For the most part I like it, but it looks like it auto-adds files as they are generated, so from now on I'll need to explicitly check and make sure to remove them.

**Pull request closed without merging**